.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
It is your project. You can accept pull requests by doing some minor reversions after.